File: quasar-front/src/app/shared/models/responses/cliente.response.ts
export class ClienteResponse {
  public codigo: number;
  public nome: string;
  public nomeFantasia: string;
  public cpfCnpj: string;
  public inscricaoEstadual: string;
  public razaoSocial: string;

  constructor(params: Partial<ClienteResponse>) {
    this.codigo = params.codigo || 0;
    this.nome = params.nome || "";
    this.nomeFantasia = params.nomeFantasia || "";
    this.cpfCnpj = params.cpfCnpj || "";
    this.inscricaoEstadual = params.inscricaoEstadual || "";
    this.razaoSocial = params.razaoSocial || "";
  }
}

File: quasar/Quasar.DataTransfer/Usuarios/Responses/ClienteResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Quasar.DataTransfer.Usuarios.Responses
{
    public class ClienteResponse
    {
        public int Codigo { get; set; }
        public string Nome { get; set; }
        public string? NomeFantasia { get; set; }
        public string? CpfCnpj { get; set; }
        public string? InscricaoEstadual { get; set; }
        public string? RazaoSocial { get; set; }        
    }
}